Sources/DDGSync/DDGSync.swift

```diff
@@ -248,6 +248,9 @@ public class DDGSync: DDGSyncing {
         dependencies.keyValueStore.set(true, forKey: Constants.syncEnabledKey)
 
         syncQueueCancellable = syncQueue.isSyncInProgressPublisher
+            .handleEvents(receiveCancel: { [weak self] in
+                self?.isSyncInProgress = false
+            })
```

Comment:
This is here to also handle cancelling subscription (e.g. when user is logged out of Sync during sync operation). Turns out that "completion" doesn't cover cancellation.